### DISCUSSION TITLE
How to change Dashboard Header Logo?

### DISCUSSION THREAD

**Discussion by Hptlive** [serenity-is/Serenity#5670] [Help] [answered]
Hi guys,
I want too know what is the best way to change logo. 

I tried following two ways, but both not workinng. Can any one please help me. 

1. Replaced the pictures in the folder \wwwroot\Content\site\images\  bu it did not work.
2. Changed the following line in file \wwwroot\Content\site\site.css but after rebuild, it again changed

      .main-header .logo i {
        width: 50px;
        height: 50px;
        float: left;
        background: url(data:image/png;base64,iVBORw0KGg.......
        }
    
  I will be thankful for a solution. 
  
  Regards

**Comment by minhhungit** [comment]
@Hptlive exactly where is the image you want to change, can you please take a screenshot, because you mention "dashboard" so I am not sure

**Comment by VictorTomaili** [comment, marked as answer]
Don't modify css files directly. Css files are compiled from .less files. Any changes on css files are overriden after build from .less files. 
Your changes must placed under this file and line https://github.com/serenity-is/Serene/blob/master/Serene/Serene.Core/wwwroot/Content/site/site.theme.less#L175

**Reply by edwardch** [reply]
And here's a handy tool to convert your images to base64: [Base64 Image Encoder](https://www.base64encode.net/base64-image-encoder)

**Reply by Hptlive** [reply]
Thank you very much.
